AnsiblePlaybooks/storage_benchmark/storage_benchmark/files/sysbench.sh

```
@@ -0,0 +1,102 @@
+#!/bin/bash
```

This file is also missing the 

```
#!/usr/bin/env bash
set -euo pipefail
```

Any step that fails will silently continue on without this for example